src/usage_metrics/helpers.py

```diff
@@ -154,3 +157,25 @@ def get_table_name_from_context(context: OutputContext) -> str:
     if context.has_asset_key:
         return context.asset_key.to_python_identifier()
     return context.get_identifier()
+
+
+def retry_request(retries=3, delay=2, backoff=2):
+    """Define a decorator to retry requests."""
```

Comment:
could you define these? they are mostly self explanatory imo except the exponential-ness of backoff